### ISSUE TITLE
locales setup on nodes

### ISSUE THREAD

**Issue by dikim33** [oscar-cluster/oscar#563]
**Reported by naughtont on 27 Jul 2009 19:50 UTC**
On the compute nodes, I'm getting language/locales warnings (see below).  There are probably several ways to fix this but it appears the following two packages "do the right thing" to generate the appropriate configs: "language-pack-en" & "language-pack-en-base"

OSCAR Version: 6.0.4 (devel)
Platform: Ubuntu 8.04.3 (LTS) on x86
System: XTORC

Issue:  Running a perl script on the compute nodes generated warning messages like the following:
```
node2:$ perl -e 'print "Hello world\n"'
perl: warning: Setting locale failed.
perl: warning: Please check that your locale settings:
        LANGUAGE = (unset),
        LC_ALL = (unset),
        LANG = "en_US.UTF-8"
    are supported and installed on your system.
perl: warning: Falling back to the standard locale ("C").
Hello world
```

Fix: Install additional packages which manage/setup locales config
```
 node2:$  sudo apt-get install language-pack-en language-pack-en-base
```

**Comment by dikim33** [comment]
**Comment by valleegr on 29 Sep 2009 06:12 UTC**
(In [8777]) This check-in fixes #563 - Avoid language/locales warnings on Ubuntu systems.

**Comment by dikim33** [comment]
**Comment by valleegr on 29 Sep 2009 06:16 UTC**
(In [8778]) This check-in fixes #563 - Includes cron in the list of package for Debian 
based images.
